File: server/src/main/scala/kpn/api/common/OrphanNodeInfo.scala
package kpn.api.common

import kpn.api.custom.Timestamp

case class OrphanNodeInfo(
  id: Long,
  name: String,
  longName: String,
  lastUpdated: Timestamp,
  proposed: Boolean,
  lastSurvey: String,
  factCount: Long,
)

File: backend/api/src/main/java/kpn/api/common/OrphanNodeInfo.java
package kpn.api.common;

import kpn.api.custom.Timestamp;

import java.util.Optional;

public record OrphanNodeInfo(
  Long id,
  String name,
  Optional<String> longName,
  Timestamp lastUpdated,
  Boolean proposed,
  Optional<String> lastSurvey,
  Long factCount
) {
}
